Source/EasyNetQ/QueueDeclareConfigurationExtensions.cs

```diff
@@ -111,5 +111,18 @@ public static IQueueDeclareConfiguration WithQueueMode(this IQueueDeclareConfigu
 
             return configuration.WithArgument("x-queue-mode", queueMode);
         }
+
+        /// <summary>
+        /// Sets queueType. Valid types are classic and quorum.
```

Comment:
```suggestion
        /// Sets queue type. Valid types are classic and quorum.
```

Reply:
The same convention is used for other parameters :(